src/Files.App/Helpers/AppThemeResourcesHelper.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Windows.UI;
+
+namespace Files.App.Helpers
+{
+	public class AppThemeResourcesHelper
```

Comment:
Use `sealed` keyword for classes as it improves performance 🤑

Reply:
Resolved